students/ShinTran/lesson03/mailroom.py

```diff
@@ -0,0 +1,106 @@
+'''
+Shin Tran
+Python 210
+Lesson 3 Assignment
+'''
+
+masterList = []
+nameList = []
+
+
+# Creates the masterList where it has 12 donations pre-populated
+def initialize():
+	global masterList
+	d1 = ["James Smith", 91561.25]
+	d2 = ["Mary Johnson", 5811.05]
+	d3 = ["John Williams", 41113.42]
+	d4 = ["Patricia Brown", 19184.81]
+	d5 = ["Robert Jones", 51227.53]
+	d6 = ["Jennifer Miller", 53514.94]
+	d7 = ["Michael David", 31051.46]
+	d8 = ["Mary Johnson", 71646.16]
+	d9 = ["William Rodriguez", 69244.21]
+	d10 = ["Mary Johnson", 12689.07]
+	d11 = ["Michael David", 57145.50]
+	d12 = ["Patricia Brown", 81679.15]
+	masterList = [d1,d2,d3,d4,d5,d6,d7,d8,d9,d10,d11,d12]
+
+# Prompts the user to enter an option
+def mainPrompt():
+	response = input("Choose from one of 3 actions:\n\
+		1) Send a Thank You\n\
+		2) Create a Report\n\
+		3) Quit\n\
+		Please type 1, 2, or 3: ")
+	return response
+
+# Takes in a user input as a parameter, enters a donation, prints a report,
+# prints list, exit, prompts again if the input is bad
+# If the user types exit it'll go back to the main prompt
+def action(userInput):
+	tempList = []
+	createNameList()
+	if userInput == 'list':
+		for i in range(0,len(nameList)):
+			print(nameList[i])
```

Comment:
please review looping over lists; this pattern is in many places. looping over list:
```
for name in name_list:
    print(name)
```